build-system/tasks/make-extension/index.js

```diff
@@ -226,6 +226,7 @@ async function makeExtensionFromTemplates(
     '__component_name_hyphenated__': name,
     '__component_name_hyphenated_capitalized__': name.toUpperCase(),
     '__component_name_pascalcase__': namePascalCase,
+    '__bento_component_name_pascalcase__': `Bento${namePascalCase}`,
```

Comment:
nit: The other keys are set without the prefix, and the prefix is usually set in code. For consistency, do this in code instead:

```js
function Bento__component_name_pascal_case__() {}
```

Reply:
I like this better, will update. Thanks!